retriever/__main__.py

```diff
@@ -178,7 +178,8 @@ def main():
                 if not searched_scripts:
                     print("No available datasets found")
                 else:
-                    print("Available offline datasets : {}\n".format(len(searched_scripts['offline'])))
+                    print("Available offline datasets : {}\n".format \
```

Comment:
some comments about this file:

- Add extra line before `def main():`

- Remove  `key = lambda` spaces

```python
                all_scripts_combined = sorted(all_scripts_combined, key=lambda x: x[0])
```
The print statement could be one line in this part of code
```python

                if not args.v:
                    print("Online datasets : {}\n".format(len(online_scripts)))
                offline_scripts = [script.name for script in all_scripts]
                for script in online_scripts:
                    if script in offline_scripts:
                        continue
                    print("{count}. {name}".format(count=count, name=script))
                    count += 1
```

And we could use variables to  store the message

 ```python
               # search
                searched_scripts = datasets(keywords, param_licenses)
                offline_mesg = "Available offline datasets : {}\n"
                online_mesg = "Available online datasets : {}\n"
                if not searched_scripts:
                    print("No available datasets found")
                else:
                    print(offline_mesg.format(len(searched_scripts['offline'])))
                    count = 1
                    for script in searched_scripts['offline']:
                        print(
                            "{count}. {title}\n{name}\n"
                            "{keywords}\n{licenses}\n".format(
                                count=count,
                                title=script.title,
                                name=script.name,
                                keywords=script.keywords,
                                licenses=str(script.licenses[0]['name']),
                            )
                        )
                        count += 1
                    print(online_mesg.format(len(searched_scripts['online'])))
                    count = 1
                    searched_scripts_offline = [script.name for script in searched_scripts['offline']]
                    for script in searched_scripts['online']:
                        if script in searched_scripts_offline:
                            continue
                        print("{count}. {name}".format(count=count, name=script))
                        count += 1
            return
```